scripts/build/builders/host.py

```diff
@@ -34,6 +34,7 @@ class HostApp(Enum):
     OTA_PROVIDER = auto()
     OTA_REQUESTOR = auto()
     PYTHON_BINDINGS = auto()
+    NL_TEST_RUNNER = auto()
```

Comment:
Why is this defined in `host.py` instead of `efr32.py`?

Reply:
This is building a host side tool to run the tests on a connected device.

Reply:
ok, thanks